pkg/types/types.go

```diff
@@ -13,21 +13,22 @@ import (
 // NetConf extends types.NetConf for ib-sriov-cni
 type NetConf struct {
 	types.NetConf
-	Master       string
-	DeviceID     string `json:"deviceID"` // PCI address of a VF in valid sysfs format
-	VFID         int
-	HostIFNames  string // VF netdevice name(s)
-	HostIFGUID   string // VF netdevice GUID
-	ContIFNames  string // VF names after in the container; used during deletion
-	GUID         string `json:"-"` // Taken from either CNI_ARGS "guid" attribute or from RuntimeConfig
-	PKey         string `json:"pkey"`
-	LinkState    string `json:"link_state,omitempty"` // auto|enable|disable
-	RdmaIso      bool   `json:"rdmaIsolation,omitempty"`
-	RdmaNetState rdmatypes.RdmaNetState
+	Master              string
+	DeviceID            string `json:"deviceID"` // PCI address of a VF in valid sysfs format
+	VFID                int
+	HostIFNames         string // VF netdevice name(s)
+	HostIFGUID          string // VF netdevice GUID
+	ContIFNames         string // VF names after in the container; used during deletion
+	GUID                string `json:"-"` // Taken from either CNI_ARGS "guid" attribute or from RuntimeConfig
+	PKey                string `json:"pkey"`
+	LinkState           string `json:"link_state,omitempty"` // auto|enable|disable
+	RdmaIso             bool   `json:"rdmaIsolation,omitempty"`
+	IBKubernetesEnabled bool   `json:"ibKubernetesEnabled,omitempty"`
+	RdmaNetState  rdmatypes.RdmaNetState
```

Comment:
gofmt error:
pkg/types/types.go:27: File is not `gofmt`-ed with `-s` (gofmt)
	RdmaNetState  rdmatypes.RdmaNetState
	Args         struct {
Makefile:74: recipe for target 'lint' failed
make: *** [lint] Error 1

Reply:
Done